src/bootstrap/bootstrap.py

```diff
@@ -952,14 +952,18 @@ def program_config(self, program, stage0=True):
 
     @staticmethod
     def get_string(line):
-        """Return the value between double quotes
+        """Return the value between double quotes inside the string with single quotes or double quotes
 
         >>> RustBuild.get_string('    "devel"   ')
         'devel'
         >>> RustBuild.get_string("    'devel'   ")
         'devel'
+        >>> RustBuild.get_string("  \"devel\"  ")
+        'devel'
         >>> RustBuild.get_string('devel') is None
         True
+        >>> RustBuild.get_string("devel") is None
```

Comment:
I think from the perspective of the function single quotes or double quotes aren't noticeable -- my recollection is that they're identical in Python, but my knowledge there might be out of date.

If I'm right, I don't think the added tests actually are any different from the existing ones, so probably we'd not want to add them.